tools/dpip/route.c

```diff
@@ -111,7 +114,13 @@ static const char *flags_itoa(uint32_t flags)
     return flags_buf;
 }
 
-static void route_dump(const struct dp_vs_route_conf *route)
+static inline bool ipv6_addr_any(const struct in6_addr *a)
```

Comment:
Use the helper function in `linux_ipv6.h`, if not exist define it there, since it's a common function.